gflownet/envs/amp.py

```diff
@@ -55,52 +79,36 @@ def __init__(
         self,
         max_seq_length=50,
         min_seq_length=1,
+        # Not required in env. But used in config_env in MLP. TODO: Find a way out
```

Comment:
Is it used to determine the dimensions of the inputs/outputs? We may want to use something like what is used in the continuous environments to determine the dimensions of the policy inputs/outputs. See [here](https://github.com/alexhernandezgarcia/gflownet/blob/continuous/gflownet/envs/htorus.py#L68)

Reply:
I see you also used these variables here, so ignore the reference. But I do think the MLP should take the dimensions in a different way.